src/librustdoc/html/markdown.rs

```diff
@@ -228,9 +228,9 @@ impl<'a, I: Iterator<Item = Event<'a>>> Iterator for CodeBlocks<'a, I> {
                 ))
             });
             let tooltip = if ignore {
-                Some(("Be careful when using this code, it's not being tested!", "ignore"))
+                Some(("This example is not tested", "ignore"))
```

Comment:
I'm not a big fan of this new sentence. :-/

Reply:
It's matter of fact, much better.